dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobProgress.java

```diff
@@ -47,6 +47,7 @@
 import lombok.*;
 import lombok.experimental.Accessors;
 import org.hisp.dhis.feedback.ErrorCode;
+import org.hisp.dhis.tracker.imports.validation.ValidationCode;
```

Comment:
not related to your changes but there are some wildcard imports in this file. maybe can clean up later.